shared/constants/chat2/index.js

```diff
@@ -202,6 +202,7 @@ export {
 
 export {
   allMessageTypes,
+  enoughTimeBetweenMessages,
```

Comment:
I'm not really sure why `enoughTimeBetweenMessages` wasn't being exported from chat2 constants but was being exported from the messages file. Since there were files that needed both `getMeta` and `enoughTimeBetweenMessages`, it made sense to add `enoughTimeBetweenMessages`!